code/game/objmob.dm

```diff
@@ -0,0 +1,2 @@
+/objmob
+	parent_type = /atom/movable
```

Comment:
Wouldn't making it `/atom/movable` make more sense?

Reply:
It's just semantics at that point, we don't write out /atom/movable/obj everywhere

Reply:
it's just literally "whatever, that's whats in this path so that's what i'll put in the PR"-tier name with zero thought behind it.

Reply:
if movable is pathed to atom then maybe /atom/movable/tangible should be as well.

Also we could put lighting objects under /atom/movable/abstract and/or move screen objects there. 

Also those are some suggested path names.